CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [7.1.0] - 2022-10-17
+
+### Added
+
+### Changed
+
+- use EAKEM (cf [MR](https://github.com/Cosmian/cover_crypt/pull/39))
```

Comment:
is there somethere the description of EAKEM you gave in this PR description? (in the README?)

Reply:
There is a work in progress on the paper side.